zipline/pipeline/pipeline.py

```diff
@@ -72,6 +77,9 @@ def add(self, term, name, overwrite=False):
             Whether to overwrite the existing entry if we already have a column
             named `name`.
         """
+        if term.ndim == 1:
+            raise UnsupportedPipelineOutput()
```

Comment:
It'd be nice for this error to tell you what the term was (the name and the value).